src/node_snapshotable.cc

```diff
@@ -840,7 +840,7 @@ size_t SnapshotSerializer::Write(const SnapshotMetadata& data) {
 // [    ...       ]  env_info
 // [    ...       ]  code_cache
 
-void SnapshotData::ToBlob(FILE* out) const {
```

Comment:
I wonder if we should now call these `SnapshotData::ToFile()`/`SnapshotData::FromFile()` which calls `ToBlob()` and `FromBlob()`..

Reply:
Sure, done in 5fa6dec34dbf059d885b074f1953943a65c74f43 :+1: